### PR TITLE
Issue 78

### PR DESCRIPTION
**Pull request by ssaru**

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -99,8 +99,8 @@ def test(params):
 
                         w_ratio = block[3]
                         h_ratio = block[4]
-                        width = int(w_ratio * W)
-                        height = int(h_ratio * H)
+                        width = int(np.sqrt(w_ratio) * W)
```
[the remark this answers is not in the document]
Comment (ssaru) [reply test.py]: 제곱이 맞나요? 아니면 sqrt가 맞나요?!

File: yolov1.py [hunk 2]
```diff
@@ -234,14 +229,17 @@ def detection_loss_4_yolo(output, target):
                                (torch.pow(width_ratio1_output - torch.sqrt(width_ratio_label), 2) +
                                 torch.pow(height_ratio1_output - torch.sqrt(height_ratio_label), 2)))
 
+    objectness_cls_map = objness_label.unsqueeze(-1)
+
+    for i in range(num_cls - 1):
+        objectness_cls_map = torch.cat((objectness_cls_map, objness_label.unsqueeze(-1)), 3)
```
[the remark this answers is not in the document]
Comment (ssaru) [reply yolov1.py]: `unzqueeze`함수는 다음과 같이 동작합니다.

예를들어 `objectness_cls_map`의 shape이 `(1,7,7)`이라고 했을 때
Loss 계산을 위해서 class의 개수만큼 `objectness_cls_map`을 stacking 해야합니다.

하지만 단순히 cat을 진행하게 되면 존재하는 `axis`에 데이터를 stacking하는 형식이라
3개의 `axis` 안에서만 선택이 가능하게됩니다.

따라서 이를 `(1,7,7,1)`로 만들어주는 함수가 `unzqueeze` 함수입니다.
